src/core/data.jl

```diff
@@ -1275,6 +1275,16 @@ function _calc_pipe_resistance(pipe::Dict{String,Any}, base_length, base_pressur
     return resistance
 end
 
+"Calculates the resistance of the well associated with as storage component"
+function _calc_well_resistance(storage::Dict{String,Any}, base_length)
+    lambda = storage["well_friction_factor"]
+    D = storage["well_diameter"]
+    L = storage["well_depth"] * base_length
+
+    resistance = (lambda * D) \ L
```

Comment:
This should be lambda * L / D

Reply:
The steady-state model is in the variable space of pressure (not density), so doesn't that make it D / L (see calc_pipe_resistance)?

Reply:
I keep forgetting. But yes, in the pressure-mass flow space, the cross-sectional area terms missing.